lib/SPIRV/libSPIRV/SPIRVValue.h

```diff
@@ -175,40 +175,59 @@ template <spv::Op OC> class SPIRVConstantBase : public SPIRVValue {
   float getFloatValue() const { return Union.FloatVal; }
   double getDoubleValue() const { return Union.DoubleVal; }
   unsigned getNumWords() const { return NumWords; }
-  SPIRVWord *getSPIRVWords() { return Union.Words; }
+  const std::vector<SPIRVWord> &getBigValWords() { return BigValWords; }
```

Comment:
It is not clear to me why this renaming is needed: I still prefer `getSPIRVWords` a s a more generic name, because you might want to do something with instructions words even if it is not a huge constant, but calling `getBigValWords` seems strange in that context.

Reply:
Reverted renaming.